packages/ui/src/__stories__/ButtonMigration/Variants.tsx

```diff
@@ -1,9 +1,115 @@
-import ButtonActionMigrationImg from '../assets/button-action-migration.png'
-import ButtonMigrationImg from '../assets/button-migration.png'
+import styled from '@emotion/styled'
+import { Snippet, Table, Text } from '../../components'
+
+const StyledCode = styled(Text)`
+  background: ${({ theme }) => theme.colors.neutral.backgroundStrong};
+  padding: ${({ theme }) => theme.space['2']};
+  border-radius: ${({ theme }) => theme.radii.default};
+  width: 100%;
+  display: block;
+`
+
+const StyledRow = styled(Table.Row)`
+  height: 70px;
+`
+
+const StyledSnippet = styled(Snippet)`
+  padding: 16px;
+`
+
+const VARIANTS_TO_SENTIMENTS = {
+  info: {
+    variant: 'filled',
+    sentiment: 'info',
+  },
+  'info-bordered': {
+    variant: 'outlined',
+    sentiment: 'info',
+  },
+  link: {
+    variant: 'ghost',
+    sentiment: 'primary',
```

Comment:
probably more a sentiment: info no ?

Reply:
@fabienhebert yes you are right I will add size also